Assets/Scripts/VrSdk.cs

```diff
@@ -743,6 +748,14 @@ private void SetUnityXRControllerStyle(InputDevice device)
             {
                 SetControllerStyle(ControllerStyle.Wmr);
             }
+            else if (device.name.StartsWith("PICO4 Touch Controller"))
+            {
+                SetControllerStyle(ControllerStyle.Phoenix);
```

Comment:
does this need to be in the `#if`?

Reply:
No, as this is just string matching. in the PICO_SUPPORTED bit below both controllers report with the same name, so that's why we needed to use the SDK instead. In their OpenXR plugin each controller has it's own name.